.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+Your description here.
+
+## Related Issues
+Closes #XXXX <- Issue number if existing
+
+## Checklist:
+<!-- Check these things before posting the pull request: -->
+- [ ]  Did you test the code?
+- [ ]  Was the Changelog Updated?
+- [ ]  Was the Documentation updated?
+- [ ]  Was the ConfigUpdater adjusted for syntax changes?
```

Comment:
To prevent spamming of `if needed` change to `Are all syntax changes accounted for in the ConfigUpdater`?